src/libraries/Microsoft.NET.WebAssembly.Webcil/src/Webcil/WebcilReader.Reflection.cs

```diff
@@ -0,0 +1,59 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Immutable;
+using System.IO;
+using System.Reflection;
+using System.Runtime.InteropServices;
+
+using System.Reflection.Metadata;
+using System.Reflection.PortableExecutable;
+
+namespace Microsoft.NET.WebAssembly.Webcil;
+
+
+public sealed partial class WebcilReader
+{
+
+    // Helpers to call into System.Reflection.Metadata internals
+    internal static class Reflection
+    {
+        private static Lazy<MethodInfo> _readUtf8NullTerminated = new Lazy<MethodInfo>(() =>
```

Comment:
`runtime` coding style - static fields should have a `s_` prefix.
```suggestion
        private static readonly Lazy<MethodInfo> s_readUtf8NullTerminated = new Lazy<MethodInfo>(() =>
```